app/src/main/java/com/codesoom/assignment/utils/JwtUtil.java

```diff
@@ -0,0 +1,56 @@
+package com.codesoom.assignment.utils;
+
+import com.codesoom.assignment.errors.InvalidTokenException;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.security.Keys;
+import io.jsonwebtoken.security.SignatureException;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.security.Key;
+
+@Component
+public class JwtUtil {
+    private final Key key;
+
+    public JwtUtil(@Value("${jwt.secret}") String secret) {
+        key = Keys.hmacShaKeyFor(secret.getBytes());
+    }
+
+    /**
+     * userId의 토큰을 생성하고 리턴한다.
+     *
+     * @param userId user의 id
+     * @return user의 토큰
+     */
+    public String encode(Long userId) {
+
+        return Jwts.builder()
+                .claim("userId", 1L)
```

Comment:
```suggestion
                .claim("userId", userId)
```

하드코딩된 코드를 수정하면 좋을 것 같아요